python/paddle/fluid/__init__.py

```diff
@@ -174,6 +174,7 @@ def __bootstrap__():
     core.init_glog(sys.argv[0])
     # don't init_p2p when in unittest to save time.
     core.init_devices(not in_test)
+    core.init_dgc()
```

Comment:
should do lazy init, not all jobs need to load this

Reply:
Done.